packages/nuxt/src/server/sdk.ts

```diff
@@ -64,3 +75,16 @@ export function mergeRegisterEsmLoaderHooks(
   }
   return options.registerEsmLoaderHooks ?? { exclude: [/vue/] };
 }
+
+/**
+ * Flushes pending Sentry events with a 2-second timeout and in a way that cannot create unhandled promise rejections.
+ */
+export async function flushSafelyWithTimeout(): Promise<void> {
```

Comment:
l: A bunch of SDKs reimplement this. Maybe we can think of hoisting it into utils? No strong feelings though.

Reply:
Good idea! But I'll create another PR for this change.